src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -201,7 +230,7 @@ CPU::CPU() {
 
     get_node_info_linux();
 
-    if (!get_cache_info_linux()) {
+    if (!get_info_linux(1)) {
```

Comment:
I think if we can change `int` value to `enum` which the meaning is more clearer.

Reply:
updated